Source/JavaScriptCore/runtime/ArrayPrototype.cpp

```diff
@@ -400,11 +403,9 @@ inline JSValue fastJoin(JSGlobalObject* globalObject, JSObject* thisObject, Stri
         auto& butterfly = *thisObject->butterfly();
         if (UNLIKELY(length > butterfly.publicLength()))
             break;
-        JSStringJoiner joiner(globalObject, separator, length);
```

Comment:
Is there a reason why we don't also `reserveCapacity` here?

Reply:
Ditto.